packages/ocsipersist/ocsipersist.1.0.1/opam

```diff
@@ -14,13 +14,13 @@ depends: [
   "dune" {>= "2.9"}
   "lwt" {>= "4.2.0"}
   "ocsigenserver" {>= "3.0.0"}
-  "ocsipersist-lib"
+  "ocsipersist-lib" {>= "1.0.0" & < "1.1.0"}
 ]
 
 depopts: [
-	"ocsipersist-dbm"
-	"ocsipersist-pgsql"
-	"ocsipersist-sqlite"
+	"ocsipersist-dbm" {>= "1.0.0" & < "1.1.0"}
+	"ocsipersist-pgsql" {>= "1.0.0" & < "1.1.0"}
+	"ocsipersist-sqlite" {>= "1.0.0" & < "1.1.0"}
```

Comment:
```suggestion
	"ocsipersist-dbm"
	"ocsipersist-pgsql"
	"ocsipersist-sqlite"
]
conflicts: [
	"ocsipersist-dbm" {< "1.0.0" | >= "1.1.0"}
	"ocsipersist-pgsql" {< "1.0.0" | >= "1.1.0"}
	"ocsipersist-sqlite" {< "1.0.0" | >= "1.1.0"}
```
constraints inside the `depopts` field does not mean the same thing as a conflict. e.g. If it is in the depopts field, if ocsipersist-dbm.1.1.0 is installed it will not be taken as a dependency but will be allowed to be installed alongside it (and will make it fail to compile)

Reply:
Thanks! Turns out I never new what depopts meant.